.github/workflows/appCI.yml

```diff
@@ -0,0 +1,18 @@
+name: appCI
+
+on: [push, pull_request]
+
+jobs:
+  build-app:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Clone Repo
+        uses: actions/checkout@v2
+      - name: Build app
+        run: |
+          cd arduinoRemote
```

Comment:
> /home/runner/work/_temp/1fae85cd-7bca-428a-94a2-7c6b22b32724.sh: line 2: ./gradlew: Permission denied

Do a `chmod +x gradlew` before running the `./gradlew build` command and you should be fine.

Reply:
Just did them in the wrong order.. Yes will try it. Thank you!

Reply:
The syntax was also slightly off and I believe they should take place in the *same* step.